dotcom-rendering/src/server/lib/get-content-from-url.js

```diff
@@ -30,6 +28,7 @@ async function getContentFromURL(url, _headers) {
 	);
 
 	// pick all the keys from the JSON except `html`
+	// eslint-disable-next-line @typescript-eslint/no-unused-vars -- we don't want `html` in the config
```

Comment:
One for us to ponder on: it might be more explicit to `delete config.html` after?